### DISCUSSION TITLE
Question and feedback

### DISCUSSION THREAD

**Discussion by gityopie** [mithnusa/odoo-google-maps#4] [Q&A]
Hello,

I would like to thank you all for your support
If you have already had a chance to install and play with the modules, please do not hesitate to ask a question or perhaps have an idea for improving the module.

Have a good day!

**Comment by iknowpc** [comment]
Possible to have Contact Image for the Icons as an option? Or at least have their contact image in the details after you click an icon? Something that won't crash the performance.

**Reply by gityopie** [reply]
Yes, it is possible. I'll have a look

I'll keep you updated and thanks for the feedback!

**Reply by iknowpc** [reply]
Another idea, is having a Web Share API button on the partner contact form.  Basically so this appears 

https://codepen.io/justforuse/pen/aboWyzR

and you can use it to send a partners address to google maps directly, or to another application.

**Reply by gityopie** [reply]
Hi @iknowpc 

I've published the update for adding contact's avatar on the contact Google Map view on the demo instance
http://35.209.158.83:8069/
user: guest
password: guest

Please have a look and let me know your feedback

**Reply by gityopie** [reply]
> Another idea, is having a Web Share API button on the partner contact form. Basically so this appears
> 
> https://codepen.io/justforuse/pen/aboWyzR
> 
> and you can use it to send a partners address to google maps directly, or to another application.

I don't this one is usable to add on the partner contact form
The administrative access is limited
unless there a link of a page on website that displayed partner contact

**Comment by iknowpc** [comment]
I logged in and it looks great. Much more visually reinforcing that
location is correct.

For the webshare it's this.
https://codeburst.io/the-web-share-api-is-here-cb651d84eccd

[image: image.png]

Basically allows a button to be pressed on a mobile device and it brings up
the different apps that share is able to process like for example going
straight to google maps or apple maps, etc...





Walter Lara
***@***.***
407-900-5565 ext.10
http://iknowpc.com

**Comment by iknowpc** [comment]
Another idea I just had, would it be possible to put a map view on tasks, so when you go to a project, or all tasks, you can see the task's contact's geo ip location, the popups could show the task name and details.
